users/views.py

```diff
@@ -1,3 +1,41 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+import bcrypt
+
+from django.core.exceptions import ValidationError
+from users.validation       import validate_email, existing_email, validate_password
+
+from django.http            import HttpResponse
+from django.views           import View
+from users.models           import User
```

Comment:
```suggestion
import json, bcrypt

from django.core.exceptions import ValidationError
from django.http            import HttpResponse
from django.views           import View

from users.validation import validate_email, existing_email, validate_password
from users.models     import User
```
import 하는 순서는 위와 같이 진행해 주세요.